generator/pom.xml

```diff
@@ -84,10 +85,12 @@
                         </goals>
                         <configuration>
                             <sourcePaths>
-                                <sourcePath>${sdk.project.dir}/src/main/resources/schema/artifact-packaged-event.json
-                                </sourcePath>
+                                <sourcePath>${parent.project.dir}/spec/schemas/artifactpackaged.json</sourcePath>
```

Comment:
The only slight issue I see with this approach is that new events will have to be added manually to this config.
It actually may be a positive thing to have a static configuration, but perhaps we should have some kind of check in place that verifies that a class has been generated for every schema available.

Reply:
Ok, I will create a separate issue to verify the number of generated files from schemas with unit testing.